### ISSUE TITLE
Fairness postprocessing technique requires complete nn output

### ISSUE THREAD

**Issue by simoneminisi** [pralab/secml-torch#28]
Some fairness postprocessing techniques (https://doi.org/10.48550/arXiv.1906.05082) require processing the entire output of the neural network. This can be problematic when the dataset is handled by a dataloader which is processed in batches (e.g., in the Accuracy metric class).
